WordPress/gutenberg
# Update the document title in the site editor to open the command tool

## What?

Now that we have a command center, we need to improve its discoverability. The idea of this PR is to update the document title a little bit to be able to trigger the command center from there. It has been tossed around several times that this area should become something close to the url bar of the browser. This is kind of a step in that direction.

<img width="326" alt="Screenshot 2023-05-05 at 10 25 06 AM" src="https://user-images.githubusercontent.com/272444/236422300-dddfa3a2-79a2-446e-8203-1a806ee31861.png">

At the moment I've implemented the following changes:

 - Allow the command center to be opened from outside (adding store actions)
 - Updating the design (simplifying) of the document title of the site editor.

The next steps will be:

 - Adapt the design of the document title when the site editor is "filled" with a page (not just an empty template), close to this:
 
![Screenshot 2023-05-05 at 11 17 04 (1)](https://user-images.githubusercontent.com/272444/236422799-142b427d-6898-4357-9600-9d58f210e8c1.png)

 - Implement contextual command center actions. (We need to specify the exact behavior here in order to come up with the right API).

**Testing instructions**

 - Navigate the site editor and check how the site editor title bar appears (template and template parts)
